code/edgemedia/src/phone/java/com/adobe/marketing/mobile/edge/media/internal/MediaTrackerEventGenerator.java

```diff
@@ -20,83 +29,21 @@
 import java.util.TimerTask;
 import java.util.UUID;
 
-class MediaTrackerEventGenerator implements MediaTracker {
-
-    private static final String EXTENSION_LOG_TAG = "Media";
-    private static final String LOG_TAG = "MediaTracker";
-    private static final String EVENT_SOURCE_TRACKER_REQUEST =
-            "com.adobe.eventsource.media.requesttracker";
-    private static final String EVENT_SOURCE_TRACK_MEDIA = "com.adobe.eventsource.media.trackmedia";
-
-    private static class EventDataKeys {
-        private EventDataKeys() {}
-
-        // Event Data Key Constants - Tracker
-        static final class Tracker {
-            private Tracker() {}
-
-            static final String ID = "trackerid";
-            static final String SESSION_ID = "sessionid";
-            static final String EVENT_NAME = "event.name";
-            static final String EVENT_PARAM = "event.param";
-            static final String EVENT_METADATA = "event.metadata";
-            static final String EVENT_TIMESTAMP = "event.timestamp";
-            static final String EVENT_INTERNAL = "event.internal";
-            static final String PLAYHEAD = "time.playhead";
-        }
-
-        // Event Data Key Constants - EventName
-        static final class MediaEventName {
-            private MediaEventName() {}
-
-            static final String SESSION_START = "sessionstart";
-            static final String SESSION_END = "sessionend";
-            static final String PLAY = "play";
-            static final String PAUSE = "pause";
-            static final String COMPLETE = "complete";
-            static final String BUFFER_START = "bufferstart";
-            static final String BUFFER_COMPLETE = "buffercomplete";
-            static final String SEEK_START = "seekstart";
-            static final String SEEK_COMPLETE = "seekcomplete";
-            static final String AD_START = "adstart";
-            static final String AD_COMPLETE = "adcomplete";
-            static final String AD_SKIP = "adskip";
-            static final String ADBREAK_START = "adbreakstart";
-            static final String ADBREAK_COMPLETE = "adbreakcomplete";
-            static final String CHAPTER_START = "chapterstart";
-            static final String CHAPTER_COMPLETE = "chaptercomplete";
-            static final String CHAPTER_SKIP = "chapterskip";
-            static final String BITRATE_CHANGE = "bitratechange";
-            static final String ERROR = "error";
-            static final String QOE_UPDATE = "qoeupdate";
-            static final String PLAYHEAD_UPDATE = "playheadupdate";
-            static final String STATE_START = "statestart";
-            static final String STATE_END = "stateend";
-        }
-
-        static final class ErrorInfo {
-            private ErrorInfo() {}
-
-            static final String ID = "error.id";
-        }
-    }
+public class MediaTrackerEventGenerator implements MediaTracker {
```

Comment:
Let's rename this class to `MediaPublicTracker` similar to iOS

Reply:
I'll leave that for future work in another PR when we add the MediaSession and MediaEventTracker classes.

Reply:
On second thought, for this one I will rename to MediaPublicTracker, as there won't be much more work on it.